ex1.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+
+
+def random_values():
+    """
+    Create a vector of length 10 with values evenly distributed between 5 and 50.
+    :return: The vector.
+    """
+    return np.random.randint(5, 55, 10)
```

Comment:
I think you misread evenly as uniformly. In that case this works great, otherwise try using linspace.
https://numpy.org/doc/stable/reference/generated/numpy.linspace.html